docs: Document how to use ListenHTTP (#765)
stackabletech/nifi-operator: docs/modules/nifi/pages/usage_guide/exposing-processors/index.adoc

= Exposing processors

> You can expose NiFi processors to the outside world using two basic mechanisms:

Reviewer: ```suggestion
There are two mechanisms to expose Nifi processors to the outside world.
```
